Setup/Uninstall.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2017-2021 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Setup;
+
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\SchemaSetupInterface;
+use Magento\Framework\Setup\UninstallInterface;
+
+/**
+ * @codeCoverageIgnore
+ */
+class Uninstall implements UninstallInterface
+{
+    /** 
+     * @param SchemaSetupInterface $setup
+     * @param ModuleContextInterface $context
+    */
+    public function uninstall(
+        SchemaSetupInterface $setup,
+        ModuleContextinterface $context
+    ){
+        $setup->startSetup();
+        // Remove Configurations 
+        $this->removeBoltConfig($setup);
+        
+        // Remove Bolt Tables
+        $tables = ['bolt_feature_switches',
+                   'bolt_external_customer_entity',
+                   'bolt_webhook_log',
+                   'bolt_customer_credit_cards'];
```

Comment:
If all the tables to remove have `bolt` in the name could we potentially get this list from the DB? Makes it so we don't have to update this if ever the DB changes or we add/remove tables.